File: PS/Day13-Attack of the Queen/Attack_of_the_Queen.cs
using System;
class s {
    static void Main() {
        int x, y, n, m, c, t = int.Parse(Console.ReadLine());
        while (t > 0) {
            string[] p = Console.ReadLine().Split();
            n = int.Parse(p[0]);
            m = int.Parse(p[1]);
            p = Console.ReadLine().Split();
            x = int.Parse(p[0]) + 1;
            y = int.Parse(p[1]) + 1;
            c = Math.Min(x - 1, y - 1) + Math.Min(n - x, m - y) + Math.Min(n - x, y - 1) + Math.Min(x - 1, m - y) + n + m - 2;
            Console.WriteLine(c);
            t--;
        }
    }
}

File: PS/Day13-Attack of the Queen/Attack_of_the_Queen.py
t=int(input())
while t>0:n,m=map(int,input().split());x,y=map(int,input().split());x+=1;y+=1;c=min(x-1,y-1)+min(n-x,m-y)+min(n-x,y-1)+min(x-1,m-y)+n+m-2;print(c);t-=1